napari/_qt/qt_resources/styles/02_custom.qss

```diff
@@ -503,17 +503,27 @@ NapariQtNotification #severity_icon {
 
 /* ------------ Plugin Dialog ------------ */
 
-QPluginList {
+QPluginList, QNotificationList {
   background: {{ console }};
 }
 
-PluginListItem {
-  background: {{ darken(foreground, 20) }};
+PluginListItem, QNotificationListItem {
+  background: "red";
   padding: 0;
   margin: 2px 4px;
   border-radius: 3px;
+  color: red;
```

Comment:
Why are those rules picked up by PluginListItem and not QNotificationListItem ?
I've tried putting duplicated rules underneath to no avail...